<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>468</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jogo da Velha</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 255);color:#ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_namegame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-weight:bold;}</string>
    </property>
    <property name="text">
     <string>Jogo da velha</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_p1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>69</y>
       <width>431</width>
       <height>160</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{color:rgb(0, 43, 129);}</string>
     </property>
     <property name="title">
      <string>Digite o nome do jogador:</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{font-weight:bold;}</string>
      </property>
      <property name="text">
       <string>Jogador 1  ( X ):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_p1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{background-color:#FFFFFF;border:0px;border-radius:5px;padding:0 10px;color:#000000;}</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_p1">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgb(0, 40, 120), stop:1 rgb(0, 62, 186));border:0px; border-radius:5px;font-weight:bold;} QPushButton:hover{background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgb(0, 22, 86), stop:1 rgb(0, 40, 120));}</string>
     </property>
     <property name="text">
      <string>Jogar</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>muz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_p2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>69</y>
       <width>431</width>
       <height>160</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{color:rgb(0, 43, 129);}</string>
     </property>
     <property name="title">
      <string>Digite o nome do jogador:</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{font-weight:bold;}</string>
      </property>
      <property name="text">
       <string>Jogador 2  ( O ):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_p2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{background-color:#FFFFFF;border:0px;border-radius:5px;padding:0 10px;color:#000000;}</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_p2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgb(0, 40, 120), stop:1 rgb(0, 62, 186));border:0px; border-radius:5px;font-weight:bold;} QPushButton:hover{background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgb(0, 22, 86), stop:1 rgb(0, 40, 120));}</string>
     </property>
     <property name="text">
      <string>Jogar</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="framegame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_c0">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c1">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c2">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c4">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>120</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c5">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>120</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c3">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>120</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c6">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>230</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c8">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>230</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_c7">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>230</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#FFFFFF; border:0px; font-size:72pt; font-weight:bold; text-align:center;color:#000000;line-height:101px;padding:10px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_informations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{text-align:center;font-size:10pt;font-weight:bold;color:rgb(255, 255, 0);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="framewin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background:rgba(0, 0, 0,90%);}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelwin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>411</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{background:RGBA(0,0,0,0%);color:rgb(255, 255, 0);font-size:40px;text-align:center;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonwin">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgb(0, 40, 120), stop:1 rgb(0, 62, 186));border:0px; border-radius:5px;font-weight:bold;} QPushButton:hover{background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgb(0, 22, 86), stop:1 rgb(0, 40, 120));}</string>
     </property>
     <property name="text">
      <string>Jogar novamente</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
